### DISCUSSION TITLE
共享农场分流流速建议

### DISCUSSION THREAD

**Discussion by cryptosoul-vioiv** [famland/website#93] [Ideas]
关于农场分流目前采用的平均共享值分配，最直接的弊端会让后续发展趋于农场单田以及30块最优农场数量。要解决这些问题，就要回归到白皮书的Famland价值打造和共享共建理念上。

为此，建议解决方案：**以农场的共享值增量决定流速。**

**具体实施为：**

1、将共享值分为基础共享值和增量共享值，即：基础共享值+增量共享值=共享值。
基础共享值是通过基本田做市产生的，增量共享值是通过野田做市转化或者通过合田产生。

2、各农场的累计增量共享值占比，决定流速的占比。

3、增量共享值为Famland的属性，加入不同农场，可为新农场带来共享值增量，提高该农场的流速占比。

4、共享值提升额度不够或者没有提升的就亮红灯，为农场主提供数据参考判断，农场主直接可以踢出，不支付任何费用。

**如此，将引导生态的发展趋势：**

1、共享值增量越大的农场，说明YL的消耗量越大，理应获利更大的流速；————按市场需求分配原则。

2、新老农场间，只要在增量上相近，那么流速就相近，能争取到的分发量也相近，不存在老玩家割新玩家的情况； 不会出现老农场一直领先，新农场追不上的情况。——符合公平公正竞争理念。

3、新农场可以通过拉更多田入场，通过数量优势与旧农场竞争流速；因此让农场主有了大量拉人进农场的驱动力，一起抱团做大共享值增量，争取更大流速；而农场单田，将在共享值增量上没有任何生存价值。——符合共享共建理念。

4、对于高增量共享值的Famland，具有很强提升农场流速的能力，将成为市场的香饽饽；挽留还是挖墙角，将成为农场主们的市场竞争游戏；Famland价值的打造，才是真正对自上而下NFT的革命。——符合价值打造理念。

5、基本共享值只是农场的入场门槛而已，不作为流速提升的要素；因为资金可以对基本共享值有绝对的控制，但在增量共享值方面将削弱话语权；基础共享值如果计算的话，在生态刚刚起步阶段，很容易被资本控制。——符合以人为本的精神。

**Comment by otto-001** [comment]
哈哈，第四点，非常喜欢。这样吭哧吭哧无脑复投的，就可以做大自己Famland的价值，走到那个农场都受欢迎。 农场主还可以给高价转会费来挖。。。😁😁✌️

**Reply by cryptosoul-vioiv** [reply]
哈哈哈，有想法。有可能激活一片新的市场出来。

**Comment by yancanlou** [comment]
以共享值增量之和来分流是个合理的分流方法，这个是坚定支持！
还要解决场内吃大锅饭问题，靠亮红灯是不行。
包括最差增量值与农场主不做为，都可以通过分类算法体现来淘汰或者不同的规则投票淘汰出局必是值得考量现实。

**Reply by cryptosoul-vioiv** [reply]
最差增量的Famland，亮红灯，可能被农场主踢出场；最差增量的农场，农场主不做为，得不到流速，场内玩家可能离场；
这就是机制下的市场淘汰行为，可以不用另用算法或投票来进行了。

**Comment by otto-001** [comment]
根据目前测试的情况，做如下几点建议：
1、
当前情况：基本田做市100U，也获得500的共享值上限，这样会引导大家100U做市基本田，然后刷野田短期内获得高额增量，抢占流速。这样大量做100U基本田+400U野田，会快速积累大量的共享值增量。抢占流速，这是不良竞争。

改进建议：基本田做市金额就是初始共享值限额。基本田做市资金全部纳入基础共享值，100—500.全部不计入增量。

做市100U，初始共享值限额就是100.
100U做市的收益比500U的少，限额提升就慢。如果二次做市基本田，那也是为了获得更多收益，更快提升共享值限额。

100U和500U做市的区别就是收益和共享值上限提升的快慢问题。玩家会自己决定100U还是500U做市

这就是基础共享值不纳入增量的意义。

所以这样，更能体现公平。投入500U满额做市基本田，肯定会比100U的收益更多，提升更快。
这样更能正面引导大家都满额做市基本田。

２、
当前情况：农场启动难。新农场进去后，没有增量，流速为0，只能在原生农场或其它农场涨增量后，才能去新建农场。

改进建议：所有新建Famland，基本田投入100U后，取1%为增量共享值。由此，在全是满额基本田做市的新农场，至少有30*5=150的增量值。

这样改进后，在农场启动的时候就不会有什么问题，就很自然的启动了。

3、
当前情况：目前实施的是增量共享值的累加占比来决定流速占比，弊端就是老农场的累加值会越来越大，可能在一个月后达到2万+的增量，而新农场增量从0开始起步，因此流速占比几乎为0，没有生存能力。

这一点，我觉得问题不大， 老农场累积的增量高，那也是一直在为生态做共享，理应获得高收益。为了保持领先，农场之间会竞争，会一直累积自己的增量。并且不是每个玩家一进场就需要建农场，都可以在原生农场或者其他农场获得收益，这些累积增量都在玩家的田里面，如果农场经营不好，玩家可以退出带走累积的增量，自己建农场或者加入其他农场。

如果改成7天平均增量的话， 所有人只看最近7天的增量，如果玩家的田累积的共享值高了，也会导致为短期增量，恶意竞争，玩家频繁进出农场。并且最近7天平均增量，这个数字是动态的，根据时间每天变化，合约实现起来也比较困难。
所以，我建议维持当前状态。
